src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@englishman I think we have to kill this PR. And you're absolutely right, that code is wrong, but can easily be fixed with `new`. Can you submit PRs for them?